lambdas/ssm-parameters/src/ssm-parameters-handler.ts

```diff
@@ -0,0 +1,38 @@
+import { LambdaInterface } from "@aws-lambda-powertools/commons";
+import { getParametersByName } from "@aws-lambda-powertools/parameters/ssm";
+import { Parameter } from "@aws-sdk/client-ssm";
+
+const PARAMETER_TTL =
+  parseNumber(process.env.POWERTOOLS_PARAMETERS_MAX_AGE) ?? 300;
+
+function parseNumber(value?: string) {
+  return parseInt(value ?? "", 10) || undefined;
+}
+export class SsmParametersHandler implements LambdaInterface {
```

Comment:
Can you look into testing the cache? You can try doing this by changing the return value and seeing if the code still returns the old one. This is the important bit because the whole point of the lambda is to fetch and cache SSM parameters.